samples/bluetooth/mesh/dfu/distributor/README.rst

```diff
@@ -219,6 +219,8 @@ When the Distributor updates itself, the DFU transfer will end immediately after
 
 When this sample is used as a Target, it behaves as described in :ref:`ble_mesh_dfu_target_upgrade`.
 
+This sample also provides support for :ref:`dfu_over_ble`, so it is possible to self-update using the Simple Management Protocol (SMP).
```

Comment:
I didn't get how distributor sample is involved into p2p dfu. Could you clarify?

Reply:
The sample allready have suppport for p2p dfu, but there is no comment about that in the documentation. This is only to give the user a hint that they also can use p2p to do self update of the distributor. One of the points in the task was to test if p2p for the distributor sample was working, and it was. So then I deceiced to mention that in the sample. Did not want to give to much documentation, as I want them to read the things in dfu_over_ble.

Reply:
If you do it this way (same for Target), you should also create the reverse references in `dfu_over_ble.rst`. Currently it specifically mentions the Light sample and Thingy53, which makes it confusing that the DFU samples also supports it but is not listed there.

Reply:
Ah, that's true, it should be mentioned in https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/protocols/bt/bt_mesh/dfu_over_ble.html#point-to_point_dfu_over_bluetooth_low_energy_in_bluetooth_mesh_samples

Reply:
Added